apolloschurchapp/Gemfile.lock

```diff
@@ -275,6 +275,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
+  x86_64-darwin-21
```

Comment:
@redreceipt This is the only change that I'm questioning. I was getting failed builds on sim when I tried to remove this. Yarn was adding this line automatically.

Reply:
yea just leave it. won't hurt. not sure why it's added sometimes.